lib_xud/doc/rst/resource_usage.rst

```diff
@@ -146,91 +151,30 @@ The ports/pins are shown in :ref:`table_xud_x200_required_pin_port`.
 
 |newpage|
 
-L-Series
-........
 
-The ports used for the physical connection to the external ULPI transceiver must
-be connected as shown in :ref:`table_xud_ulpi_required_pin_port`.
-
-.. _table_xud_ulpi_required_pin_port:
-
-.. table:: ULPI required pin/port connections
-    :class: horizontal-borders vertical_borders
-
-    +-------+-------+------+-------+---------------------+
-    | Pin   | Port                 | Signal              |
-    |       +-------+------+-------+---------------------+
-    |       | 1b    | 4b   | 8b    |                     |
-    +=======+=======+======+=======+=====================+
-    | X0D12 | P1E0  |              | ULPI_STP            |
-    +-------+-------+------+-------+---------------------+
-    | X0D13 | P1F0  |              | ULPI_NXT            |
-    +-------+-------+------+-------+---------------------+
-    | X0D14 |       | P4C0 | P8B0  | ULPI_DATA[7:0]      |
-    +-------+       +------+-------+                     |
-    | X0D15 |       | P4C1 | P8B1  |                     |
-    +-------+       +------+-------+                     |
-    | X0D16 |       | P4D0 | P8B2  |                     |
-    +-------+       +------+-------+                     |
-    | X0D17 |       | P4D1 | P8B3  |                     |
-    +-------+       +------+-------+                     |
-    | X0D18 |       | P4D2 | P8B4  |                     |
-    +-------+       +------+-------+                     |
-    | X0D19 |       | P4D3 | P8B5  |                     |
-    +-------+       +------+-------+                     |
-    | X0D20 |       | P4C2 | P8B6  |                     |
-    +-------+       +------+-------+                     |
-    | X0D21 |       | P4C3 | P8B7  |                     |
-    +-------+-------+------+-------+---------------------+
-    | X0D22 | P1G0  |              | ULPI_DIR            |
-    +-------+-------+------+-------+---------------------+
-    | X0D23 | P1H0  |              | ULPI_CLK            |
-    +-------+-------+------+-------+---------------------+
-    | X0D24 | P1I0  |              | ULPI_RST_N          |
-    +-------+-------+------+-------+---------------------+
-
-In addition some ports are used internally when the XUD library is in
-operation. For example pins X0D2-X0D9, X0D26-X0D33 and X0D37-X0D43 on
-an XS1-L device should not be used.
-
-Please refer to the device datasheet for further information on which ports
-are available.
-
-
-Core Speed
-------------
+Core Frequency
+--------------
 
 Due to I/O requirements, the library requires a guaranteed MIPS rate to
 ensure correct operation. This means that core count restrictions must
 be observed. The XUD core must run at at least 80 MIPS.
 
 This means that for an xCORE device running at 500MHz there should be no more
-than six cores executing at any time when using the XUD. xCORE-200 devices all setting
+than six cores executing at any time when using the XUD. Devices all setting
```

Comment:
The text starting with 'Devices all setting' is not a complete sentence.

Reply:
Fixed. Thanks.